app/src/main/res/values/strings.xml

```diff
@@ -95,13 +95,13 @@
 
     <string name="pref_sync_fblink" translatable="false">pref_sync_fblink</string>
     <string name="pref_sync_fblink_title">Add link to Facebook</string>
-    <string name="pref_sync_fblink_summary">Add a link to Facebook to event descriptions.</string>
+    <string name="pref_sync_fblink_summary">Add a link to event description on Facebook.</string>
```

Comment:
Not quite sure about this.

Reply:
It's in the "Configure Sync Preferences" menu. When the option is checked, it will append a link to the event on facebook.com to the event description, so your change is incorrect. I realize the original string is a little awkward, I don't know how to express it better.